Scripts/fastlane/ScreenshotFastfile

```diff
@@ -0,0 +1,144 @@
+    locales = {
+      "da" => {},
+      "de-DE" => {},
+      "en-AU" => {},
+      "en-CA" => {},
+      "en-GB" => {},
+      "en-US" => {},
+      "es-ES" => {},
+      "fr-FR" => {},
+      "id" => {},
+      "it" => {},
+      "ja" => {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60},
+      "ko" => {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60},
+      "no" => {},
+      "nl-NL" => {},
+      "pt-BR" => {},
+      "pt-PT" => {},
+      "ru" => {},
+      "sv" => {},
+      "th" => {},
+      "tr" => {},
+      "zh-Hans" => {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60},
+      "zh-Hant" => {font: "/Library/Fonts/Arial Unicode.ttf", text_size: 60}
+    }
+
+    config_data = [
+      {device: "iPhone 8 Plus", template: "./fastlane/appstoreres/assets/ios-phone-5-5-template.png", comp_size: "924x1640", comp_offset: "+161+568"},
+      {device: "iPhone X", template: "./fastlane/appstoreres/assets/ios-phone-5-8-template.png", comp_size: "833x1803", comp_offset: "+146+632", text_offset: 90},
+      {device: "iPad Pro (12.9-inch)", template: "./fastlane/appstoreres/assets/ios-tab12-landscape-template.png", comp_size: "2298x1723", comp_offset: "+221+322", text_offset: 20, text_adj: 80}
+    ]
+
+    orig_folder = "./screenshots_orig"
+    output_folder = "./screenshots"
+
+    # Set up folders
+    if (File.exists?(output_folder) && !File.exists?(orig_folder)) then
+      if UI.confirm("Do you want to use the files in #{output_folder} as originals?")
+        FileUtils.mv(output_folder, orig_folder)
+        Dir.mkdir(output_folder)
+      end
+    end 
+
+    # Run screenshots generator tool 
+    promo_screenshots(orig_folder: "./fastlane/screenshots_orig",
+      output_folder: "./fastlane/screenshots",
+      locales: locales,
+      device_config: config_data,
+      default_locale: "en-US", 
+      metadata_folder: "./fastlane/appstoreres/metadata")  
+  end 
+
+  #####################################################################################
+  # download_promo_strings
+  # -----------------------------------------------------------------------------------
+  # This lane downloads the promo strings to use for the creation of the enhanced 
+  # screenshots.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane download_promo_strings 
+  #
+  # Example:
+  # bundle exec fastlane download_promo_strings
+  #####################################################################################
+  desc "Downloads translated promo strings from GlotPress"
+  lane :download_promo_strings do |options| 
+    files = {
+      "app_store_screenshot-1" => {desc: "app_store_screenshot_2.txt"},
+      "app_store_screenshot-2" => {desc: "app_store_screenshot_5.txt"},
+      "app_store_screenshot-3" => {desc: "app_store_screenshot_3.txt"},
+      "app_store_screenshot-4" => {desc: "app_store_screenshot_1.txt"},
```

Comment:
Is it intentional that these numbers don't match up?

Reply:
Yes, it is. 
The left value is the GlotPress key, so the translations get lost when it changes.
The right value is that target file name. The index in this name is related to the order of the screenshots in the Store. 

Screenshots used to have a certain order, then we changed it, but we didn't change the content of the messages, so the translations are still good. 
We use this hash to map every key to the right file (i.e. the screenshot it the right position).